src/main/java/com/zendesk/maxwell/MaxwellFilter.java

```diff
@@ -130,6 +149,24 @@ public boolean matches(String database, String table) {
 		return matchesDatabase(database) && matchesTable(table);
 	}
 
+	private boolean matchesValues(Map<String, Object> data) {
+		for (Map.Entry<String, String> entry : includeColumnValues.entrySet()) {
+			String column = entry.getKey();
+
+			if (data.containsKey(column)) {
+				String columnValue = entry.getValue();
+
+				Object value = data.get(column);
+				if (value == null) return false;
+
+				String valueString = value.toString();
+				if (!columnValue.equals(valueString)) return false;
+			}
+		}
+
+		return true;
```

Comment:
I guess the decision to return true if the row doesn't contain the column is fine, but we should be explicit about it in the documentation.

Reply:
Yes, if none of the columns is found in the row, it returns true.